src/Microsoft.Extensions.PlatformAbstractions/DefaultRuntimeEnvironment.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Extensions.PlatformAbstractions
+{
+    public class DefaultRuntimeEnvironment : IRuntimeEnvironment
+    {
+        public DefaultRuntimeEnvironment()
+        {
+            OperatingSystem = GetOs();
+            RuntimePath = GetLocation(typeof(object).GetTypeInfo().Assembly);
+            RuntimeType = GetRuntimeType();
+            RuntimeVersion = typeof(object).GetTypeInfo().Assembly.GetName().Version.ToString();
+            RuntimeArchitecture = GetArch();
+        }
+
+        // TODO: implement
```

Comment:
Going to fix this in the PR or file a bug for later? Either way, we need to track this